### ISSUE TITLE
Unable to install packages

### ISSUE THREAD

**Issue by Muirey03** [wstyres/Zebra#120]
**Describe the bug**
When I attempt to install packages, I get the following console output and the tweak will not be installed:
```
Downloading Packages.
Downloading XXX
Installing Packages...
Importing local packages.
Finished importing local packages.
Done!
```

**To Reproduce**
For me, I just need to try to install any new package, for example:
1. Search for "Beta Alert" (not installed)
2. Press "Beta Alert!"
3. Press "Install"
4. Press "Confirm"

**Expected behavior**
The tweak should be installed

**Screenshots**
Here is a screenshot of the console output: https://imgur.com/a/MtNZ049

**Device Information (please complete the following information):**
 - iOS Version [e.g. 12.1.2 or 10.3.3]: 12.1.1
 - Zebra Version [e.g. 1.0\~beta1 or beta1]: 1.0~beta11
 - Device Type [e.g. iPhone or iPad]: iPhone

**Crash Log or System Log**
N/A

**Additional context**
This has been happening ever since I installed Zebra a few betas ago and I haven't been able to fix it

**Comment by PoomSmart** [comment]
Does this just happen to your for every beta of Zebra? Are there any additional steps that can always reproduce this issue? @Muirey03

**Comment by Muirey03** [comment]
@PoomSmart Just downgraded it to 1.0~beta2 to test, and yes this bug still happens. As for the additional steps, I don't know. This happens for me every time I try to install anything.

**Comment by PoomSmart** [comment]
@Muirey03 Can you try compiling Zebra from this repository and test?

**Comment by Muirey03** [comment]
@PoomSmart I would, as there's other features and stuff that I'd like to play around with, but I do all my development on windows so can't build xcode projects :(

**Comment by datitisev** [comment]
This happens to me as well. I can try compiling Zebra and testing it.

**Comment by datitisev** [comment]
After building it myself, I can install packages just fine...  package name is the same.

Debug build works just fine.

After installing the release build, I see "Clear Image Cache" and "Clear Keychain", so I definitely have the latest version (not sure why I didn't see it in debug build). It still installs the packages as it should.

I tested it with NoBlur from your repo.

**Comment by Muirey03** [comment]
@wstyres @PoomSmart I have worked out exactly why this issue is happening, prepare for my long and complicated explanation:

- The issue is caused when you have 1 tweak with `com.apple.UIKit` in the bundle filter, and another with `xyz.willy.Zebra` in the bundle filter (the two tweak I used to test were "Clean Tab" and "TweakReviewsDB", but there are probably others causing this issue)
- The reason this happens is because supersling is located in the same bundle as Zebra
- This means that TweakReviewsDB is loaded into supersling when it shouldn't be
- TweakReviewsDB links against UIKit, meaning UIKit is also loaded into supersling
- This means that all UIKit tweaks (including CleanTab) are loaded into supersling
- This breaks supersling

The best and most universal fix is this: Move supersling out of Zebra's bundle. Cydo is also not located in Cydia's bundle for presumably this same reason. Nothing should ever be injected into supersling, and certainly not tweaks like TweakReviewsDB intended for Zebra so moving it outside of the bundle makes perfect sense.

If you have any queries or want me to test anything for you, hit me up.

**Comment by wstyres** [comment]
I’m not able to reproduce this with TweakReviewsDB and Clean Tab installed. Does uninstalling either one fix the issue for you / are you sure both of these are the culprit?

**Comment by Amplespencer** [comment]
Going to uninstall TweakReviewsDB and report back

**Comment by Amplespencer** [comment]
nvm just forgot Im not jailbroken and can't be rejailbroken until i get a pc because ignition/jailbreak.fun isn't working for me (popup won't show)

**Comment by Muirey03** [comment]
@Amplespencer don't worry about it. Wilson was able to reproduce this issue on his unc0ver device so a fix should be coming soon.

**Comment by Amplespencer** [comment]
ok, I just really hope Letmeblock didn't break or something because I don't
get popups from signing services after I restored rootfs :/

**Comment by wstyres** [comment]
Should be fixed as of 02882d4
